Get package from local instead of URL

hanryhu opened the pull request:
Use package_dir instead of a URL which is no longer open to public use
or is out-of-date.  Fixes Issue https://github.com/google-research/lottery-ticket-hypothesis/issues/2.

hanryhu committed:
Get package from local instead of URL

Use package_dir instead of a URL which is no longer open to public use
or is out-of-date.

mapleyuen96 commented:
hello,i have run setup.py with your help successully,thank you very much.
But after that,when i try to run the download_data.py,it comes an error:ModuleNotFoundError:No Module named 'lottery_ticket'. I have checked the code,and i have never seen the method to import like that in this code.Is it correctly？How to fix it?
please help me 
thanks again!

hanryhu committed:
Make various small fixes

There were some problems which were small but nevertheless prevented
the correct running of the code.  Now, a `python setup.py install` and
a `pip install .` in the main directory should do the trick.

Errors included IO-related stuff, missing arguments, and for some
reason packages not being installed correctly.

hanryhu commented:
@mapleyuen96 I was using this code for a project and figured out how to get it to run.  I don't know much about python modules but this commit seems to do the trick.  Main thing is I did `pip3 install .` in the main directory.

Another thing is, my `python` isn't `python3` by default, so I had to switch all appearances of `"python"` in the scripts to instead be `"python3"`.

Enjoy!

mapleyuen96 commented:
> @mapleyuen96 I was using this code for a project and figured out how to get it to run. I don't know much about python modules but this commit seems to do the trick. Main thing is I did `pip3 install .` in the main directory.
> 
> Another thing is, my `python` isn't `python3` by default, so I had to switch all appearances of `"python"` in the scripts to instead be `"python3"`.
> 
> Enjoy!

thank you for kindly replying!
I have solved the problem by 'unzip xxx.egg'(xxx.egg is generated by setup.py,i think) in python/site-packages.Although i do not know the reason,but it does work.

Howerver,when i try to run the` runner/lottery_experiment.py`,other bugs occur.
1. Initial component
2. ('The function received no value for the required argument:', 'output_dir')

I noticed that `mnist/lottery_experiment.py` defines a `train` function,but i haven't found where to define the variable ‘output_dir’. Is it defined in a certain .py file?Or could i just insert a line of code like "output_dir = './xxx/xxx' "before using the train func?
Please give me your advice.Also,my english is not very good.If you are good at Chinese,please answer me in Chinese so that I can understand you better.
Thank you very much!
